lib/cylc/gui/gscan.py

```diff
@@ -1002,7 +1002,7 @@ def _update_group_counts(self):
                 # Compat:<=7.5.0
                 group_id = suite_info.get(KEY_GROUP)
             # For the purpose of this method, it is OK to handle both
-            # witheld (None) and unset (empty string) together
```

Comment:
@kinow - are you a genius-level English-as-a-second-language speller, or is your IDE helping you out here? Just curious...

Reply:
Definitely not genius. Most typos were found with PyCharm or mwic (interesting little python project... groups typos by number of occurrences.... showing less frequent at the top).

Some were found when I was reading the code and trying to write tests, or just following the logic in the code. Many comments saved me minutes of debugging and trial and error coding!

Reply:
Great ... I tend to miss subtle-ish mistakes like "witheld" when reading at speed. :grimacing: